### DISCUSSION TITLE
showDebugInformation=true not showing debug footer or link to view routes

### DISCUSSION THREAD

**Discussion by cfcoder** [wheels-dev/wheels#1345] [Q&A] [answered]
What am I doing wrong? I have the line:

`set(showDebugInformation = true);`

in my config/development/settings.cfm file but my app footer still does not show any debug footer.

**Comment by bpamiri** [comment]
@cfcoder You shouldn't need this setting in development mode as the debug footer should show by default in development mode. 

config > development > setting.cfm
```
<cfscript>
// Use this file to configure specific settings for the "development" environment.
// A variable set in this file will override the one in "config/settings.cfm".

// Example:
// set(dataSourceName="devDB");
set(showDebugInformation = true);
</cfscript>
```

This shouldn't have any effect, but setting it to false as in:
```
<cfscript>
// Use this file to configure specific settings for the "development" environment.
// A variable set in this file will override the one in "config/settings.cfm".

// Example:
// set(dataSourceName="devDB");
set(showDebugInformation = false);
</cfscript>
```
Should hide the debug information.
![image](https://github.com/cfwheels/cfwheels/assets/180555/fe7b5bf3-bde2-4413-8d15-f072639418f2)

**Reply by bpamiri** [reply]
Also keep in mind you can always get back to the congratulations page with the debug links across the top by visiting `/wheels/wheels` in your browser.

**Reply by bpamiri** [reply]
Lastly, make sure your `config > environment.cfm` is set accordingly:
```
<cfscript>
// Use this file to set the current environment for your application.
// You can set it to "development", "testing", "maintenance" or "production".
// Don't forget to issue a reload request (e.g. reload=true) after making changes.
// See http://docs.cfwheels.org/docs/switching-environments for more info.

// Below, we have set it to "development" for you since that is convenient when you are building your application.
// We recommend that you change this to "production" when you're running your application live.
set(environment = "development");
</cfscript>
```

**Comment by cfcoder** [comment]
I think the issue has to do with a setting I found in the html tag for hidding the overflow-y:
![image](https://github.com/cfwheels/cfwheels/assets/4595721/756ebbb2-10d0-4807-bae2-c86214f81873)

When I remove that style attribute, I now have the problem that the link to view the routes is not showing as your screenshot shows.
![image](https://github.com/cfwheels/cfwheels/assets/4595721/54469a94-caf9-4a55-b3d6-19c32e82717e)

**Comment by bpamiri** [comment]
Can you add the following to see what it give:
```
<cfdump var="#$get("enablePluginsComponent")#">
```

**Reply by cfcoder** [reply]
It gives "true"

**Comment by neokoenig** [comment, marked as answer]
Judging by your screenshot, you're in `design` mode which was deprecated ages ago.
Try ?reload=development&password=whatever.

**Reply by neokoenig** [reply]
Technically you can do `?reload=whatever` and it will attempt to load an evironment for it.

**Reply by cfcoder** [reply]
What is the difference between "design" and "development"? I compare the two settings.cfm files for each and they look the same (in my situation).

**Reply by neokoenig** [reply]
design was dropped ages ago (c. wheel 2.x) 
I can't remember what the actual difference was, I think it was that model config() (or rather back then, init()) methods were cached in one and not the other and it was just confusing.

**Comment by cfcoder** [comment]
Yay! Development mode shows the links including for the view of the Routes. Thank you!
